flags/application.json

```diff
@@ -14,6 +14,26 @@
     "shift": 4,
     "undocumented": true
   },
+  "RPC_PRIVATE_BETA": {
+    "description": "Application has access to the RPC HTTP server",
+    "shift": 5,
+    "undocumented": true
+  },
+  "ALLOW_ASSETS": {
+    "description": "Relates to RPC application assets",
+    "shift": 8,
+    "undocumented": true
+  },
+  "ALLOW_ACTIVITY_ACTION_SPECTATE": {
+    "description": "Whether the application can allow users to spectate a game from RPC",
```

Comment:
For consistency with my prior comment:
`"Enables activity spectating for the application",`